testsuite/tests/lib-unix/common/test_unix_cmdline.ml

```diff
@@ -31,27 +31,25 @@ all_modules= "test_unix_cmdline.ml"
 
 *)
 
-open Unix
-
 let prog_name = "cmdline_prog.exe"
 
 let run args =
-  let out, inp = pipe () in
-  let in_chan = in_channel_of_descr out in
+  let out, inp = Unix.pipe () in
```

Comment:
Out of curiosity, is there a strong reason for these changes? (removal of `open Unix` and adding `Unix.` qualifiers all over the place)

Reply:
grep'ing, as in the commit message - there were many more instances of qualified uses of `Unix`, that commit just made the tests consistently not use `open Unix` (or aliasing, but that wasn't used anyway) which meant I could see every use of `Unix.environment` or `Unix.getenv` with a simple fgrep...